lang/po/Firmware_fr.po

```diff
@@ -296,44 +296,44 @@ msgstr "Change correctement?"
 #: ../../Firmware/messages.cpp:21 ../../Firmware/ultralcd.cpp:5838
 #: ../../Firmware/ultralcd.cpp:6860
 msgid "Checking X axis"
-msgstr "Verific. axe X"
+msgstr "Vérifier axe X"
 
 #. MSG_CHECKING_Y c=20
 #: ../../Firmware/messages.cpp:22 ../../Firmware/ultralcd.cpp:5848
 #: ../../Firmware/ultralcd.cpp:6861
 msgid "Checking Y axis"
-msgstr "Verific. axe Y"
+msgstr "Vérifier axe Y"
 
 #. MSG_SELFTEST_CHECK_Z c=20
 #: ../../Firmware/ultralcd.cpp:6862
 msgid "Checking Z axis"
-msgstr "Verific. axe Z"
+msgstr "Vérifier axe Z"
 
 #. MSG_SELFTEST_CHECK_BED c=20
 #: ../../Firmware/messages.cpp:94 ../../Firmware/ultralcd.cpp:6863
 msgid "Checking bed"
-msgstr "Verif. plateau"
+msgstr "Vérifier lit"
 
 #. MSG_SELFTEST_CHECK_ENDSTOPS c=20
 #: ../../Firmware/ultralcd.cpp:6859
 msgid "Checking endstops"
-msgstr "Verific. butees"
+msgstr "Vérifier butées"
 
 #. MSG_CHECKING_FILE c=17
 #: ../../Firmware/ultralcd.cpp:6955
 msgid "Checking file"
-msgstr "Verific. fichier"
+msgstr "Vérifier fichier"
 
 #. MSG_SELFTEST_CHECK_HOTEND c=20
 #: ../../Firmware/ultralcd.cpp:6865
 msgid "Checking hotend"
-msgstr "Verif. du hotend"
+msgstr "Vérifier du hotend"
 
 #. MSG_SELFTEST_CHECK_FSENSOR c=20
 #: ../../Firmware/messages.cpp:95 ../../Firmware/ultralcd.cpp:6866
 #: ../../Firmware/ultralcd.cpp:6867
 msgid "Checking sensors"
-msgstr "Verif. des capteurs"
+msgstr "Vérifier d. capteurs"
```

Comment:
Could simplify with "Vérifier capteurs". The "d." is clearly so uncommon, it doesn't have any meaning for us.